### PR TITLE
[PT-BR] Translate node meetups

### PR DESCRIPTION
**Pull request by maugravena**
waiting for review @nodejs/nodejs-pt

### CODE REVIEW

**Comment by alexandrtovmach** [comment]
@nodejs/nodejs-pt

**Comment by khaosdoctor** [comment]
Looking into it

**Review by khaosdoctor — APPROVED** [review]

File: locale/pt-br/get-involved/node-meetups.md [hunk 1]
```diff
@@ -0,0 +1,671 @@
...
+- Contato do organizador - [@maudelaquintana](https://twitter.com/maudelaquintana)
+
+### Brasil
+
+#### São Paulo
+
+- [Meetup](https://meetup.com/nodebr)
+- Frequência dos meetups - mensal
+- Como enviar uma palestra? Fale com os organizadores na página do meetup
+- Nomes dos organizadores - [Erick Wendel](https://github.com/erickwendel), [Alan Hoffmeister](https://github.com/alanhoff), [Igor França](https://github.com/horaddrim), [Icaro Caldeira](https://github.com/icarcal), [Marcus Bergamo](https://github.com/thebergamo), [Igor Halfeld](https://github.com/igorHalfeld).
```
Comment (khaosdoctor) [inline locale/pt-br/get-involved/node-meetups.md]:
```suggestion
- Nomes dos organizadores - [Erick Wendel](https://github.com/erickwendel), [Alan Hoffmeister](https://github.com/alanhoff), [Igor França](https://github.com/horaddrim), [Icaro Caldeira](https://github.com/icarcal), [Marcus Bergamo](https://github.com/thebergamo), [Igor Halfeld](https://github.com/igorHalfeld), [Lucas Santos](https://github.com/khaosdoctor).
```

Just updating here as I did on #2690

File: locale/pt-br/get-involved/node-meetups.md [hunk 2]
```diff
@@ -0,0 +1,671 @@
...
+- Contato do organizador - [@erickwendel_](https://twitter.com/erickwendel_)
```
Comment (khaosdoctor) [inline locale/pt-br/get-involved/node-meetups.md]:
```suggestion
- Contato do organizador - [@erickwendel_](https://twitter.com/erickwendel_), [@_StaticVoid](https://twitter.com/_staticvoid)
```

Updating in favor of #2690

**Review by khaosdoctor — CHANGES_REQUESTED** [review]
Just update the organisers as I did in #2690 because otherwise we'd have to create another PR just for this

**Review by khaosdoctor — APPROVED** [review]

**Comment by khaosdoctor** [comment]
Hey @maugravena can you please check the errors in the CI [here](https://github.com/nodejs/nodejs.org/pull/2683/checks?check_run_id=263066313)? 
 Just open the "test" and there are some markdown lint messages

**Comment by maugravena** [comment]
Thanks @khaosdoctor

**Review by khaosdoctor — APPROVED** [review]

**Comment by khaosdoctor** [comment]
Ok, so now thinks should be working @Trott @MaledongGit another one to be merged!

**Comment by alexandrtovmach** [comment]
@maugravena Thanks for contribution
@khaosdoctor Thank you for quick review

**Pull request merged**